homeassistant/components/dynalite/bridge.py

```diff
@@ -0,0 +1,123 @@
+"""Code to handle a Dynalite bridge."""
+import pprint
+
+from dynalite_devices_lib import DynaliteDevices
+from dynalite_lib import CONF_ALL
+
+from homeassistant.core import callback
+from homeassistant.const import CONF_HOST
+
+from .const import DOMAIN, DATA_CONFIGS, LOGGER
+from .light import DynaliteLight
+
+
+class BridgeError(Exception):
+    """Class to throw exceptions from DynaliteBridge."""
+
+    def __init__(self, message):
+        """Initialize the exception."""
+        super().__init__()
+        self.message = message
+
+
+class DynaliteBridge:
+    """Manages a single Dynalite bridge."""
+
+    def __init__(self, hass, config_entry):
+        """Initialize the system based on host parameter."""
+        self.config_entry = config_entry
+        self.hass = hass
+        self.area = {}
+        self.async_add_entities = None
+        self.waiting_entities = []
+        self.all_entities = {}
+        self.config = None
+        self.host = config_entry.data[CONF_HOST]
+        if self.host not in hass.data[DOMAIN][DATA_CONFIGS]:
+            LOGGER.info("invalid host - %s", self.host)
+            raise BridgeError("invalid host - " + self.host)
```

Comment:
Please use string formatting.
```suggestion
            raise BridgeError(f"invalid host - {self.host}")
```

Reply:
done. thanks